pkg/operator/resources/cluster/controller.go

```diff
@@ -110,6 +110,10 @@ func getControllerClusterPolicyRules() []rbacv1.PolicyRule {
 			},
 			Verbs: []string{
 				"get",
+				"list",
+				"watch",
```

Comment:
See comment about RBAC above, would love to just add create instead of the rest as well.

Reply:
see above

Reply:
So I checked https://github.com/kubevirt/containerized-data-importer/blob/main/pkg/operator/resources/namespaced/controller.go#L66-L76 and we already have `*` on config maps in our namespace. What exactly is not working for you that you need this extra permission? @mhenriks am I being too paranoid with list/watch on config maps?

Reply:
Switched to `uncachedClient.Get()` and removed the list/watch rbac.

Reply:
https://github.com/kubevirt/containerized-data-importer/blob/80ac74bd9cf3a5026a8852752152fb33d1a31c69/cmd/cdi-controller/controller.go#L345
So read once=>cached=>needs list/watch

I also struggle to answer if list/watch are fine here, whether they are cheap or not, etc
But since it's always a GET I think we won't be hitting the API server too hard so uncached is also fine